### ISSUE TITLE
Using groundtruth during testing

### ISSUE THREAD

**Issue by yaosx425** [Luo-Z13/pointobb#19]
Hello author! I used a script to test and then prompted me to enter information such as groundtruth. Why does the test also require annotations such as groundtruth?

**Comment by Luo-Z13** [comment]
> Hello author! I used a script to test and then prompted me to enter information such as groundtruth. Why does the test also require annotations such as groundtruth?

Hello, please refer to https://github.com/Luo-Z13/pointobb/blob/main/PointOBB/README.md. The inference process of PointOBB generates rotated box pseudo labels from point labels in the training set, so point annotations are still required during the inference process.

**Comment by yaosx425** [comment]
> > Hello author! I used a script to test and then prompted me to enter information such as groundtruth. Why does the test also require annotations such as groundtruth?
> 
> Hello, please refer to https://github.com/Luo-Z13/pointobb/blob/main/PointOBB/README.md. The inference process of PointOBB generates rotated box pseudo labels from point labels in the training set, so point annotations are still required during the inference process.

Hello,author! Because I trained my own annotated dataset to change it to the type of DOTA, how do I test it?
